app/models/student.rb

```diff
@@ -8,6 +8,7 @@ class Student < ActiveRecord::Base
 
   belongs_to :homeroom, optional: true, counter_cache: true
   belongs_to :school
+  belongs_to :student_photo
```

Comment:
I'd prefer to model this as "student has one student photo." That feels more logical and natural-language to me: you'd naturally say that a student has a photo of him/her, but you wouldn't say that a photo has a student. Here's the Rails guide on this one: http://guides.rubyonrails.org/association_basics.html#choosing-between-belongs-to-and-has-one.

Reply:
Yep!  The idea previously was to store all new photos we get.  They each have a reference to the student, so we'd store that (a foreign key).  Separately, we also want to know what the most recent photo for the student is, so that's what this was (another foreign key).  So that data has meaning separate from how the Rails language is on top.  So `current_student_photo` might have been a more accurate name previously.

And cool to do whatever you think is best to 🚢 this!

Reply:
Cool, thanks for the explanation!